cpu/stm32l1/periph/i2c.c

```diff
@@ -116,9 +116,9 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed)
         /* disable peripheral */
         i2c->CR1 &= ~I2C_CR1_PE;
         /* toggle both pins to reset analog filter */
-        _toggle_pins(port, pin_scl, pin_sda);
+        _toggle_pins(dev);
```

Comment:
@PeterKietzmann Okey, seems to be working without it and I couldn't find something in the errata sheet.